integrationtests/server_test.go

```diff
@@ -0,0 +1,201 @@
+package integrationtests
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"io/ioutil"
+	"math/big"
+	mrand "math/rand"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/lucas-clemente/quic-go/h2quic"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("Server tests", func() {
+	mrand.Seed(time.Now().UnixNano())
+
+	var (
+		port    string
```

Comment:
This variable is shadowing the global `port` we currently have in the tests. That won't be an issue if we restructure it out later, but for now I'd like to avoid that. Rename to `serverPort`?